app/controllers/taxonomy_todos_controller.rb

```diff
@@ -7,12 +7,16 @@ def show
   end
 
   def update
-    todo_form = TaxonomyTodoForm.new(todo_params)
-    todo_form.taxonomy_todo = taxonomy_todo
-    todo_form.user = current_user
-    todo_form.save
-
-    redirect_to_next_item
+    @todo_form = TaxonomyTodoForm.new(todo_params)
```

Comment:
I would expect to see some tests in a PR so we can guarantee that we are not breaking anything.